index.html

```diff
@@ -60,12 +60,10 @@
       text="Learn more about who is taking your money and how to build power to fight back."
       target=urls.power-report.endpoint -%}
 
-    {% assign image_url =
-      urls.dispute-tools.endpoint | append: "/images/tools/hand.png" %}
     {% assign dispute_url =
       urls.dispute-tools.endpoint | append: urls.dispute-tools.paths.dispute-tools %}
     {%- include learn-more.html heading="Fight Back"
-      image=image_url image_size="77%" image_position="50% 58%"
+      image="assets/images/fist.png" image_size="77%" image_position="50% 58%"
```

Comment:
let's fix the other paths in this file to look like this line (without the `./` at the front),  I don't think that works for HTML. ex: https://github.com/debtcollective/landingpage/blob/0c884fff94cf93b07ad8ea61ff051def1f44a4c4/index.html#L28-L29.

Reply:
The `./` definitely does works in the HTML spec, but it should probably be an absolute path with just a leading `/`.

Reply:
I think we need these as relative because of the github domain (https://debtcollective.github.io/landingpage), and it will work when we change domains.

Reply:
Oh rats, that's right!

Reply:
`./` is a relative path and it doesn't cause issues (afaik). I'll change it, but I have a question: are you asking for style reasons or is there a technical reason?

Reply:
code style